### DISCUSSION TITLE
Bug or expected behaviour?

### DISCUSSION THREAD

**Discussion by tegitting** [nicotine-plus/nicotine-plus#2874] [Q&A] [answered]
When I browse my own shares I am not able to view the contents of child directories as the cursor/selection keeps jumping to the parent (main) directory.

Turning on debugging shows that every 1 second that it appears that I am browsing my own shares over and over and over (when I am simply just viewing them and trying to see the contents of child directories). Every second is when the cursor jumps. This does not feel like expected behaviour but I thought it best to ask before filing a bug.

Debug shows this:
12/02/24 19:38:13 User XxxXXXxxX is browsing your list of shared files
12/02/24 19:38:14 User XxxXXXxxX is browsing your list of shared files
12/02/24 19:38:15 User XxxXXXxxX is browsing your list of shared files
12/02/24 19:38:15 User XxxXXXxxX is browsing your list of shared files
12/02/24 19:38:16 User XxxXXXxxX is browsing your list of shared files
12/02/24 19:38:17 User XxxXXXxxX is browsing your list of shared files

**Comment by mathiascode** [comment, marked as answer]
That's not intended behavior. The log message isn't supposed to appear when you browse your own shares either.

Are you on Nicotine+ 3.3.0? Do you have any plugins enabled?

**Reply by tegitting** [reply]
Hi. Yes I am on 3.3.0 and have leech detector enabled.

**Comment by tegitting** [comment]
It seems to be something with the addons, I will try and find out what I have did wrong. Sorry to have jumped on the discussion thread. I am new to this client and still working things out.

**Reply by slook** [reply]
> Sorry to have jumped on the discussion thread.

No need to apologize, you are very welcome to post in here.

> I will try and find out what I have did wrong.

Have you been able to confirm if it is the Leech Detector plugin that leads to the errors, or if it is a bug in Nicotine+ which may need further investigation?
